### ISSUE TITLE
add support for envoy 1.18.2 in hubble-ui

### ISSUE THREAD

**Issue by aanm** [cilium/cilium#15842]
It's not currently possible to run hubble-ui with envoy 1.18.2 as it returns the following error:
```
[2021-04-23 02:43:39.135][1][info][main] [source/server/server.cc:815] exiting
The v2 xDS major version is deprecated and disabled by default. Support for v2 will be removed from Envoy at the start of Q1 2021. You may make use of v2 in Q4 2020 by following the advice in https://www.envoyproxy.io/docs/envoy/latest/faq/api/transition. (Unknown field in: {"static_resources":{"clusters":[{"connect_timeout":"0.25s","name":"frontend","type":"strict_dns","lb_policy":"round_robin","hosts":[{"socket_address":{"address":"127.0.0.1","port_value":8080}}]},{"type":"logical_dns","lb_policy":"round_robin","connect_timeout":"0.25s","name":"backend","hosts":[{"socket_address":{"port_value":8090,"address":"127.0.0.1"}}],"http2_protocol_options":{}}],"listeners":[{"filter_chains":[{"filters":[{"config":{"stat_prefix":"ingress_http","codec_type":"auto","http_filters":[{"name":"envoy.filters.http.grpc_web"},{"name":"envoy.filters.http.cors"},{"name":"envoy.filters.http.router"}],"route_config":{"name":"local_route","virtual_hosts":[{"cors":{"expose_headers":"grpc-status,grpc-message","max_age":"1728000","allow_methods":"GET, PUT, DELETE, POST, OPTIONS","allow_headers":"keep-alive,user-agent,cache-control,content-type,content-transfer-encoding,x-accept-content-transfer-encoding,x-accept-response-streaming,x-user-agent,x-grpc-web,grpc-timeout","allow_origin_string_match":[{"prefix":"*"}]},"name":"local_service","domains":["*"],"routes":[{"route":{"prefix_rewrite":"/","max_grpc_timeout":"0s","cluster":"backend"},"match":{"prefix":"/api/"}},{"route":{"cluster":"frontend"},"match":{"prefix":"/"}}]}]}},"name":"envoy.filters.network.http_connection_manager"}]}],"address":{"socket_address":{"port_value":8081,"address":"0.0.0.0"}},"name":"listener_hubble_ui"}]}})
```

**Comment by aanm** [comment]
Marking as release blocker since the version used in the helm charts is extremely old and outdated.

**Comment by joestringer** [comment]
Fixed by #15847
